pkgs/applications/gis/gmt/default.nix

```
@@ -106,7 +106,7 @@ in stdenv.mkDerivation (finalAttrs: {
     '';
     platforms = lib.platforms.unix;
     license = lib.licenses.lgpl3Plus;
-    maintainers = with lib.maintainers; [ tviti ];
+    maintainers = with lib.maintainers; teams.geospatial.members ++ [ tviti ];
```

```suggestion
    maintainers = with lib.maintainers; lib.teams.geospatial.members ++ [ tviti ];
```

otherwise it doesn't evaluate

```
nix eval --json  .#gmt.meta.maintainers
error: undefined variable 'teams'

       at /nix/store/6pyc572h1q17qad930nabkfn9sbj318y-source/pkgs/applications/gis/gmt/default.nix:109:41:

          108|     license = lib.licenses.lgpl3Plus;
          109|     maintainers = with lib.maintainers; teams.geospatial.members ++ [ tviti ];
             |                                         ^
          110|   };
```

---

Updated, and I unified all the ways of declaring maintainers.